First draft of article

abachma2 opened the pull request:
This is the first full draft of my article for the EPJ-N special issue. They have stated that the article should be 4-15 pages. In it's current state, I am at 21 pages. If i remove the extra spacing then I am at 17 pages. The instructions for authors (https://www.epj-n.org/author-information/instructions-for-authors) state to have tables and figures at the end, and I am working on getting that done, but I wanted to let you have the draft at least. 

Current pdf of the draft: 
[main.pdf](https://github.com/arfc/2021-bachmann-epjn/files/6777308/main.pdf)

abachma2 committed:
added Bachmann and Huff acknowledgements

abachma2 committed:
added Madicken, fixed spelling

abachma2 committed:
added more sources, objective to intro, main methods

abachma2 committed:
added more about recipes and materials

abachma2 committed:
First draft of intro, added acros, outline for results and conclusions

abachma2 committed:
removed LWR only scenario

abachma2 committed:
updated methodology, added trademark for MMR

abachma2 committed:
added energy and reactor deployment figures

abachma2 committed:
added energy figures and some results for Scenario 1

abachma2 committed:
adjsuted X-energy capitalization

abachma2 committed:
added some results discussion for Scen. 1,2

abachma2 committed:
added figures for each scenario

abachma2 committed:
inserted figures for fuel mass and swu results

abachma2 committed:
added details of archetypes and enrichment feed

abachma2 committed:
added some results discussion

abachma2 committed:
added some results discussion for energy and reactors

abachma2 committed:
added results discussion

abachma2 committed:
corrected figure to use for SWU, added more results discussion

abachma2 committed:
reordered background info of intro

abachma2 committed:
added more results discussion

abachma2 committed:
completed discussion, needs edits

abachma2 committed:
finished abstract

abachma2 committed:
edited to be in the word limit

abachma2 committed:
added thanks to Nathan

abachma2 committed:
wrote quick conclusion and some ideas of future work

abachma2 committed:
adjusted corresponding author format

abachma2 committed:
adjusted labels and sizing of the figures

abachma2 committed:
removed some acronyms

abachma2 committed:
fixed y axis label for consistency

nsryan2 commented:
[Ab_Epjn_2021_07_review_nr.pdf](https://github.com/arfc/2021-bachmann-epjn/files/6786254/Ab_Epjn_2021_07_review_nr.pdf)
I tried to keep all direct edits to be in red, and suggestions in blue. Let me know if there's anything else I can help out with!
As far as the page limit goes, I don't know if I cut out two pages worth of text. It should be a little shorter with these suggestions though.

abachma2 committed:
text revisions from @nsryan2

abachma2 committed:
adjusted figure spacing

abachma2 committed:
removed commented package

abachma2 committed:
adjsuted figure size and spacing

abachma2 committed:
adjusted some wording for consistency

abachma2 commented:
I fixed the grammar issues @nsryan2 identified. He mentioned which affiliation we should use for @munkm (NPRE or NCSA) and if there are any acknowledgements for her we need to include. 
Updated draft:
[main.pdf](https://github.com/arfc/2021-bachmann-epjn/files/6786967/main.pdf)

munkm reviewed (commented):
This is round 1 of my review. I haven't finished reading it but I will do my best to get to the rest tomorrow during lunch. This is a well-written, very thorough writeup of your work so far. Very nice job. 

**Minor Comments:**

Line 5 : New reactors are likely of an advanced reactor design --> New reactors are likely to be an advanced reactor design
Line 11 : “and may limit new reactor design deployment” – maybe use delay rather than limit here, since much of your earlier dialogue is time-focused
Line 16: and down-blending and the enrichment of NU – add a comma between down-blending and and so it’s clear these are two separate methods.
Line 45: “This work aims to quantify the material resources of the front end of the transition to different types of HALEU-fueled advanced reactors, assuming that NU will be enriched to produce fuel for LWRs and HALEU-fueled reactors.” – maybe rephrase this to something like “This work aims to quantify the material resource requirements at the front end of the transition for different types of HALEU-fueled advanced reactors, assuming that NU will be enriched to produce fuel for LWRs and HALEU-fueled reactors.”
Line 54: “Materials of interest include” – maybe “Metrics of interest”?
Your introduction into Table 1 is very nice.
l112: really nice introduction into the results!
the x axis caption is cut off in Fig 3
What does the peak in Fig 3 in ~2016 correspond to?
l133: “and a maximum of 3.95×106 kg-SWU” – do you mean kg-SWU/mo?
l137: “deployment schedule of the LWRs” – maybe revise to “the deployment and decommissioning schedule of the LWRs”.
Figure 5 is very nice. It may be helpful to make the far right y-axis the same maroon color as the line you’ve used for the number of reactors since it corresponds to that.
The subfigures in Figure 6 should be increased in size, if possible, and the axes labels should be enlarged.
Most of your figure sentences start with “figure x shows…”. There is some repitition and you may want to come up with alternative phrasings like “illustrates”, etc.
l148 “are clearly some time periods” – remove clearly here.
if needed, you can cut the sentence “This causes the large, sudden increases in uranium sent to these reactors and the periods of no uranium sent to them.”. If you do decide to keep it, make a note of the periodicity of this demand for Uranium and how predictable it can be.

**Longer Comments:**

Line 21: “Understanding the material requirements of the transition to HALEU-fueled reactors will provide insight into how each of these methods can be used to obtain the HALEU fuel needed to meet electricity demands.” – you also have an important point here that these material requirements may provide useful insight into the limitations on the speed with which a HALEU transition can occur. I think that’s an important point to add here.

Line 30: “used on fuel characteristics such as the type of fuel in the reactors (uranium, U/Th), neutron spectrum, and the inclusion of spent fuel recycling.” - are these some of the 9 metrics mentioned in the previous sentence? If so, I might rephrase these two to have the first sentence say a few metrics and include this list. If not, then I would say that you should switch to talk about the study categorization first and then mention the metrics over which each group was evaluated. That way you’re describing the problem “setup” first and the varying variable second.

If you need to cut pages, the paragraph staring at line 33 and going to 44 is very good, but not super critical to the understanding of your work.

Line 79: “including the mass required for each refueling, was obtained from [7, 8].” – I think this does work, but it might be nice to have an additional descriptor of these sources here. Consider something like “was obtained from supplementary sources [7, 8]”.

Paragraph beginning at l90: consider adding a sentence here about whether you have included license extensions in this work.

l100: "Fuel cycle facilities included in the simulations range from the uranium mine to the final disposal of the spent fuel. " --> “This simulation modeled the fuel cycle from the uranium mine to the final disposal of spent fuel.”

l118: you mention that the last LWR is decommissioned in 2016. Do you mean decommissioning is finished in 2016? How do LWRs produce power into 2025 if that’s true? I think some clarifications would be helpful here.

**Other:**
Your plots should have similar axes label sizes as the text in your paper. If you have time, you could consider scaling them a bit. However, I do not think this is an urgent change as they are readable in their current form.

abachma2 committed:
added NCSA for Madicken, weird formatting

abachma2 committed:
made revisions based on comments from @munkm

abachma2 committed:
revisions from @munkm's comments

abachma2 committed:
changed font sizes of figures

abachma2 committed:
made wording revisions

abachma2 committed:
tried to get NCSA affiliation to show properly, not working

abachma2 commented:
I made some more edits, and changed it to the preprint mode, which condensed the text and moved the figures (but not the tables) to the end of the document. I added the NCSA affiliation for Madicken, but it doesn't show the correct letter and I can't figure out how to fix that. 

Updated pdf:
[main.pdf](https://github.com/arfc/2021-bachmann-epjn/files/6818784/main.pdf)

munkm commented on authors.tex:
```diff
@@ -1,6 +1,7 @@
+\ead{amandab7@illinois.edu}
+\author[uiuc]{Kathryn D. Huff\corref{corrauthor}}
+\ead{kdhuff@illinois.edu}
+\author[uiuc, ncsa]{Madicken Munk}
```
I swear you're going to hate this, but

```suggestion
\author[uiuc,ncsa]{Madicken Munk}
```

abachma2 replied:
Oh my gosh. You're right; I do hate this.

munkm reviewed (commented):
Ok here is the rest of the review, but I used the PDF I started reviewing with, so the line numbers are based on that. Really nice work! Let me know if the suggestion I gave fixes the affiliation for you. It does for me locally. 

Minor comments:
- l145: "the energy 145 produced by LWRs decreases to 89.35 GWe-y in 2030" -- here I'd add a clause saying what the demand is supposed to be, so the deficit and it's quantity is clear to the reader before they see Fig 5. 
- l158: "This causes the large, sudden increases in uranium sent to these reactors and the periods of no uranium sent to them." -- 
- l162: "but the maximum value is 267.71 MTU more than the maximum mass of enriched uranium sent to the LWRs in Scenario 1" --> the maximum value exceeds the maximum mass of enriched uranium sent to LWRs in Scenario 1 by 267.71. 
- l164: I am not sure how this is a separate paragraph from the previous one? It seems they belong together since they're both describing Fig 6 and Scenario 2. Consider either combining them or adding an introductory sentence to this paragraph that distinguishes it from the previous one. 
- l184: I really like this paragraph detailing the differences between Scenario 2 and 3. 
- Do you know why the Figure 9 caption text is smaller than the subfigure caption text? 
- l198: consider combining this paragraph with the previous one. You're describing figure 9's subfigures and I think they're topically similar enough to be in the same paragraph.
- l203: "There is no substantial change in the SWU capacity needed to enrich uranium for the LWRs in Scenario 1 and for the Xe-100 reactors in this scenario, despite the higher enrichment level of uranium sent to the Xe-100 reactors." This reads a bit confusingly to me. Maybe you could rephrase to something like "..no substantial change in the SWU capacity needed to enrich uranium between Scenarios 1 and 3, despite.."? Or say "between a LWR-only and a Xe-100 transition scenario". I think because you're talking about Scenario 3 it might be clearer to do the former. 
- l210: "These values are comparable to what is required to enrich uranium in Scenario 1." -- this sort of repeats what you already said in the paragraph. You could remove it and not lose clarity. 
- l213: "of the scenario" -- you could say "of the scenario with 1% growth" to remind your reader of how this scenario differs from 2. 
- l215: You mention that there isn't a huge deficit in supply/demand here even though there was in Scenario 2. When did the 1% growth start in your models? I think a reader migth wonder why the demand isn't met in a no-growth scenario but is met better in a 1% scenario. Adding commentary on why this would happen would be helpful. 
- l215: you say there is a small deficit in this scenario, remove small. 
- l225: remove "unsurprisingly" 
- I am really liking how as you build up the scenarios you compare the results to previous scenarios. Very nice discussion.
- l231: "Figure 13 shows the SWU capacity required to enrich uranium for all the
reactors in the scenario, and only for the MMRTMs." -- This reads a bit oddly without seeing the figure knowing that it has two subfigures. I think adding clarity to this sentence would make it smoother. 
- sentence between l232 and 235 - this is a very similar sentence as what you said in your description for Scenario 2. Consider saying "For the same reasons described in Scenario 2, the SWU required to enrich U for MMRs is larger than LWRs" so that you're avoiding repeating exactly what you said already
- l239 - "but slightly higher than the average SWU capacity required to enrich uranium
240 for the LWRs in Scenario 1.". Do you mean before 2025? Or average for all of Scenario 1? It seems like they are substantially higher than the average of 0.74×106 kg-SWU/month and maximum of 3.95×106 kg-SWU for the LWRs. 
- l287: "The transition scenarios exhibit some discrepancies in energy demand and produced, localized to the beginning of the transition scenarios, and to the replacement of MMRTMs as they are decommissioned." --> "The transition scenarios exhibit some gaps between energy production and demand, localized to the beginning of the transition scenarios, and to the replacement of MMRTMs as they are decommissioned."
- l290: "because they have a much longer lifetime than the MMRTMs." -- Are they not simulated because they have a longer lifetime than MMRs or because their lifetime exceeds the time span simulated? 
- l293: VERY NICE conclusion paragraph here. 
- l293: "The results of this work show" --> "This work shows that.."
- l302: Add an introductory sentence to this section and then mention a couple of examples of future work. 


Other Comments:
- if you're looking to continue reducing length, you could add the max and average values of SWU and U mass to the subfigure captions for each of your scenarios. You have many sentences like "Enriching uranium for the Xe-100 reactors requires an average of 0.82×106 kg-SWU/timestep and a maximum of 3.64×106 kg-SWU" that could be removed if you do this and then you could make the statements comparing the values to other transition scenarios within the text itself.

katyhuff commented:
Huff review in #2 .

abachma2 committed:
fixed author affiliation

abachma2 committed:
removed Huff acknowledgements

abachma2 committed:
changed title

abachma2 committed:
revised based on comments from @katyhuff

abachma2 committed:
changes based on comments from @munkm and @katyhuff

abachma2 committed:
removed DOE discussion

abachma2 committed:
adjsuted figure sizes

abachma2 commented:
I have addressed all of the comments from @munkm and @katyhuff (via Issues #2). Here is the final draft: 
[main.pdf](https://github.com/arfc/2021-bachmann-epjn/files/6823428/main.pdf)

abachma2 committed:
fixed typo

abachma2 committed:
fixed another typo

nsryan2 commented on conclusion.tex:
```diff
@@ -1,2 +1,41 @@
+on-line refueling, so fuel is continually sent to them. 
+
+\section{Future Work}
+One possible are of future work is extending the end date 
```
```suggestion
One possible area of future work is extending the end date 
```

nsryan2 commented on abstract.tex:
```diff
@@ -1,7 +1,25 @@
+To understand the material requirements of enriching natural uranium to 
+produce \gls{HALEU}, this 
+work simulates multiple fuel cycle scenarios to compare how the size of 
+advanced reactor deployed and the energy growth demand affect the 
```
```suggestion
the advanced reactor deployed and the energy growth demand affect the 
```

nsryan2 commented on abstract.tex:
```diff
@@ -1,7 +1,25 @@
+material requirements of the transition to \gls{HALEU}-fueled reactors. Fuel 
+cycle scenarios considered include a no-growth and a 1\% growth transition to 
+either a microreactor fleet or a small modular reactor fleet from the 
+current fleet of Light Water Reactors.The Ultra Safe Nuclear Coropration 
```
```suggestion
current fleet of Light Water Reactors. The Ultra Safe Nuclear Corporation 
```

nsryan2 commented on introduction.tex:
```diff
@@ -1,10 +1,72 @@
+and the enrichment of \gls{NU}. Downblending \gls{HEU} is 
+limited by the physical inventory of \gls{HEU} available. 
+Enriching \gls{NU} is limited by the amount of uranium that can be 
+mined and \gls{SWU} capacity to enrich the urnaium. Understanding the 
```
```suggestion
mined and the \gls{SWU} capacity required to enrich the uranium. Understanding the 
```

nsryan2 commented on introduction.tex:
```diff
@@ -1,10 +1,72 @@
+on the type of reactor, enrichment level, and transition scenario.
+Therefore, it is important to accurately model the transition to any future
+reactor designs to quantify the material requirements and understand
+if current supply chain is sufficient to meet them. 
```
```suggestion
if the current supply chain is sufficient to meet them. 
```

nsryan2 commented on methods.tex:
```diff
@@ -1,23 +1,138 @@
+\cite{huff_fundamental_2016}.
+
+The scenarios model the fuel cycle from the uranium mine to the final 
+disposal of the fuel in the HLW Sink.  Figure 
```
```
disposal of the fuel in the HLW Sink.  Figure 
```
Expand HLW

nsryan2 commented on methods.tex:
```diff
@@ -1,23 +1,138 @@
+The spent \gls{LWR} fuel 
+recipe assumes a 51 MWd/kg-U burnup. All other recipes capture the 
+necessary isotopic ratios of uranium, but do not include other elements. 
+The enrichment feed recipe assume natural uranium is used, and the tails 
```
```suggestion
The enrichment feed recipe assumes natural uranium is used, and the tails 
```

nsryan2 commented on results.tex:
```diff
@@ -1,34 +1,435 @@
+correspond to the decommissioning of \glspl{MMR} at the end of their lifetime 
+and the deployment of new reactors, such as from 2062-2069. 
+The deployment of \glspl{MMR} in 2031 contributes to the inability to 
+meet the energy demand of the scenario between 203-2050, because the energy 
```
```suggestion
meet the energy demand of the scenario between 2030-2050, because the energy 
```

nsryan2 commented on results.tex:
```diff
@@ -1,34 +1,435 @@
+just the \glspl{MMR}. There is a large difference in the \gls{SWU} 
+capacity needed to enrich uranium for \glspl{LWR} and \glspl{MMR}. This 
+is because the \glspl{MMR} require a much larger mass and 
+a higher enrchment level. An average of 1.37$\times 10^6$ kg-\gls{SWU}/month
```
```suggestion
a higher enrichment level. An average of 1.37$\times 10^6$ kg-\gls{SWU}/month
```

nsryan2 commented on results.tex:
```diff
@@ -1,34 +1,435 @@
+the energy produced by each reactor type, and the energy demand. 
+Xe-100 reactors are deployed in October 2031, which is the same as when 
+\glspl{MMR}
+are deployed in Scenario 2. A maximum of 1225 Xe-100 reactors are deployed.
```
```suggestion
are deployed in Scenario 2. Scenario 3 deploys a maximum of 1,225 Xe-100 reactors.
```

nsryan2 commented on results.tex:
```diff
@@ -1,34 +1,435 @@
+
+\subsection{Scenario 4}
+Figure \ref{fig:energy_rx_4} shows the number of \glspl{MMR} deployed, 
+energy produced by each type of reactor, and the energy demand of this
```
```suggestion
the energy produced by each type of reactor, and the energy demand of this
```

nsryan2 commented on results.tex:
```diff
@@ -1,34 +1,435 @@
+scenario. The Xe-100 reactors are deployed in January 2030, the 
+same as when 
+\glspl{MMR} are deployed in Scenario 4. The maximum number of Xe-100 
+reactors deployed in the scenario is 2339. 
```
```suggestion
reactors deployed in the scenario is 2,339. 
```

nsryan2 commented on results.tex:
```diff
@@ -1,34 +1,435 @@
+reactors. The \gls{SWU} capacity needed 
+to enrich uranium for the Xe-100 reactors starts out at a similar 
+amount as the \glspl{LWR}, but 
+increases as the enrgy demand and the number of reactors increases. 
```
```suggestion
increases as the energy demand and the number of reactors increases. 
```

nsryan2 reviewed (commented):
Just a few minor things I noticed!

nsryan2 commented on institutions.tex:
```diff
@@ -1 +1,2 @@
 \address[uiuc]{Dept. of Nuclear, Plasma, and Radiological Engineering, University of Illinois at Urbana-Champaign, Urbana, IL 61801}
+\address[ncsa]{National Center for Supercomputing Applications, University of Illinios at Urbana-Champaign, Urbana, IL, 61801}
```
```suggestion
\address[ncsa]{National Center for Supercomputing Applications, University of Illinois at Urbana-Champaign, Urbana, IL, 61801}
```

nsryan2 commented on introduction.tex:
```diff
@@ -1,10 +1,72 @@
-All tables and figures must be referenced to in the text.
-Use the BibTeX package to manage your paper references and bibliography
-\cite{huff_extensions_2014}.
+Most of the \glspl{LWR} operating the U.S. are slated to retire
```
```suggestion
Most of the \glspl{LWR} operating in the U.S. are slated to retire
```

nsryan2 commented on methods.tex:
```diff
@@ -1,23 +1,138 @@
+requiring \gls{HALEU} for fuel, then quantifies and compares the front-end 
+material requirements of each scenario. Five different fuel cycle scenarios 
+are modeled using \Cyclus \cite{huff_fundamental_2016};
+an open source, agent-based fuel cycle simulator. The first scenario models 
```
```suggestion
an open-source, agent-based fuel cycle simulator. The first scenario models 
```

nsryan2 commented on methods.tex:
```diff
@@ -1,23 +1,138 @@
+grid connection date. Only reactors with a power level above 400 MWe are included 
+to ensure that no research or experimental reactors are used. 
+The mass of fuel in the \gls{LWR} reactor cores, including the mass  
+required for each refueling, was obtained from supplementary soources 
```
```suggestion
required for each refueling, was obtained from supplementary sources 
```

nsryan2 commented on methods.tex:
```diff
@@ -1,23 +1,138 @@
+\cite{todreas_nuclear_2012,cacuci_handbook_2010}.
+All \glspl{LWR} are assumed to have an 18 month cycle length. 
+
+We used a variety of open source documents to obtain data about the advanced reactors
```
```suggestion
We used a variety of open-source documents to obtain data about the advanced reactors
```

nsryan2 commented on methods.tex:
```diff
@@ -1,23 +1,138 @@
+at the power produced in 2025. The energy demand of each transition scenario 
+(Scenarios 2-5) is modeled by either a linear (for no growth) or 
+exponential (for 1\% growth) equation defined by the \Cycamore GrowthRegion
+archetype \cite{huff_fundamental_2016}. The \glspl{LWR} are depoyed 
```
```suggestion
archetype \cite{huff_fundamental_2016}. The \glspl{LWR} are deployed 
```

nsryan2 commented on results.tex:
```diff
@@ -1,34 +1,435 @@
-\end {table}
+\subsection{Scenario 1}
+The amount of energy supplied by the \glspl{LWR} and the number of \glspl{LWR}
+deployed as a function of time is shown in Figure \ref{fig:energy_rx_1}. 
```
```suggestion
deployed as a function of time are shown in Figure \ref{fig:energy_rx_1}. 
```

nsryan2 commented on results.tex:
```diff
@@ -1,34 +1,435 @@
+\gls{LWR} is deployed in June of 2016 and decommissioned in July 2076. The 
+maximum number of 
+\glspl{LWR} deployed at one time is 109. The energy produced by the 
+\glspl{LWR} follows with the number of reactors deployed. The maximum energy 
```
```suggestion
\glspl{LWR} follows the number of reactors deployed. The maximum energy 
```

nsryan2 commented on results.tex:
```diff
@@ -1,34 +1,435 @@
+The deployment and decommissioning schedule of the \glspl{LWR} in this 
+scenario is 
+applied to each of the other scenarios, and therefore the material 
+requirements of each of the following scenarios is the same as those 
```
```suggestion
requirements of each of the following scenarios are the same as those 
```

nsryan2 commented on results.tex:
```diff
@@ -1,34 +1,435 @@
+than
+average mass sent to the \glspl{LWR}. The lower average is explained by the 
+multiple time periods in 
+which no \gls{HALEU} is sent to the \glspl{MMR} due to the lack of refuleing.  
```
```suggestion
which no \gls{HALEU} is sent to the \glspl{MMR} due to the lack of refueling.  
```

nsryan2 commented on results.tex:
```diff
@@ -1,34 +1,435 @@
+is because the \glspl{MMR} require a much larger mass and 
+a higher enrchment level. An average of 1.37$\times 10^6$ kg-\gls{SWU}/month
+and a maximum of 20.3$\times 10^6$ kg-\gls{SWU}
+is needed to enrich the uranium sent to \glspl{MMR}. These values are both 
```
```suggestion
are needed to enrich the uranium sent to \glspl{MMR}. These values are both 
```

nsryan2 reviewed (commented):
A couple spelling and verb things that I missed.

abachma2 committed:
spelling and grammar fixes from @nsryan2

abachma2 commented:
Made the revisions from @nsryan2. I submitted the draft on July 15, but these fixes will be incorporated into the revisions.

abachma2 commented:
Most recent pdf: 
[main.pdf](https://github.com/arfc/2021-bachmann-epjn/files/6901177/main.pdf)
